react/index.js

```diff
@@ -66,12 +66,12 @@ export class MiniCart extends Component {
     })
   }
 
-  get itemsQuantity() {
+  itemsFiltered = () => {
```

Comment:
Why remove get? With the get, you can call this way `this.itemsFiltered`

Reply:
Hmmm, this may be personal but I hate this `get`. When reading code through `render()` I think it is weird and misleading that `this.itemsFiltered` is in fact calling a function. Using `this.itemsFiltered()` is much better. @lbebber agrees with me and I believe Breno commented he doesn't like `get` as well.

If you really want I can keep the get though, no problem!

Reply:
(just perhaps rename the function to `getFilteredItems`)